File: app/src/test/java/com/ccg/plat/VideoListBean.java
package com.ccg.plat;


import com.ccg.plat.entity.VideoBean;

import java.util.List;

/**
 * @author : C4_雍和
 * 描述 :
 * 主要功能 :
 * 维护人员 : C4_雍和
 * date : 20-9-4 下午2:04
 */
public class VideoListBean {
    private List<VideoBean> data;
    private String videoTag;

    public List<VideoBean> getData() {
        return data;
    }

    public void setData(List<VideoBean> data) {
        this.data = data;
    }

    public String getVideoTag() {
        return videoTag;
    }

    public void setVideoTag(String videoTag) {
        this.videoTag = videoTag;
    }
}

File: app/src/main/java/com/ccg/plat/entity/VideoListBean.kt
package com.ccg.plat.entity

data class VideoListBean(
    val `data`: MutableList<Data>,
    val videoTag: String
) {
    data class Data(
        val e: String,
        val i: String,
        val id: Int,
        val name: String,
        val pUrl: String,
        val tags: String,
        val url: String,
        val vUrl: String
    )
}